webapp/src/main/java/org/opentestsystem/rdw/admin/model/Group.java

```diff
@@ -0,0 +1,78 @@
+package org.opentestsystem.rdw.admin.model;
+
+public class Group {
+    private String name;
+    private String schoolName;
+    private String subject;
```

Comment:
That reminds me I wanted to but did not complete remove the rdw-admin-common module, I mentioned this to Brandon.